examples/window.rs

```diff
@@ -1059,6 +1069,10 @@ const KEY_BINDINGS: &[Binding<&'static str>] = &[
         Action::AnimationCustomCursor,
     ),
     Binding::new("Z", ModifiersState::CONTROL, Action::ToggleCursorVisibility),
+    // T
+    Binding::new("T", ModifiersState::empty(), Action::SetTheme(None)),
+    Binding::new("T", ModifiersState::SHIFT, Action::SetTheme(Some(Theme::Light))),
+    Binding::new("T", ModifiersState::CONTROL, Action::SetTheme(Some(Theme::Dark))),
```

Comment:
`T` is a bit unfortunate, because it's used for macOS tabs, and just `Shift+T` is also unfortunate, because it'll block regular input of `T`.

Reply:
Do you have a better suggestion?

Reply:
Any other letter which is not over used? The help is printed anyway, so which letter it is doesn't really matter.